packages/react-components/src/organisms/SharedResearchOutputBanners.tsx

```diff
@@ -43,7 +43,8 @@ const SharedResearchOutputBanners: React.FC<
     setReviewBannerState(
       reviewToggled ? (reviewRequestedBy ? 'requested' : 'dismissed') : null,
     );
-  }, [reviewToggled, reviewRequestedBy]);
+    setPublishedNowBanner(publishedNow);
```

Comment:
is this line required? I wonder if it is because we are already setting `publishedNowBanner` as `publishedNow` here

```
const [publishedNowBanner, setPublishedNowBanner] = useState(publishedNow);
```

right?